pyartcd/pyartcd/pipelines/quay_doomsday_backup.py

```diff
@@ -31,10 +30,10 @@ def __init__(self, runtime: Runtime, all_arches: bool, arches: Optional[list[str
         if not (arches or all_arches):
             raise Exception("Either --arches or --all-arches needs to be specified")
 
-        self.arches = arches.split(",") if not all_arches else self.ALL_ARCHES_LIST
+        self.arches = arches.split(",") if not all_arches else ALL_ARCHES_LIST
```

Comment:
```suggestion
        self.arches = arches if arches else ALL_ARCHES_LIST
```
make sure to look at what the type is being passed. Your IDE should have complained though, did you not do a trial run after this code change?

Reply:
I did do a trial run and everything worked as intended since the only issue was the incorrect type of `arches` in the definition of `QuayDoomsdaySync.__init__()` which does not affect the actual functionality.
But it is weird that my IDE did not complain about this type mismatch, I'll have a look later at whether I have misconfigured something in the settings